File: json/json-annotation-processor/src/test/java/ru/tinkoff/kora/json/annotation/processor/dto/DtoWithSnakeCaseNaming.java
package ru.tinkoff.kora.json.annotation.processor.dto;

import ru.tinkoff.kora.common.NamingStrategy;
import ru.tinkoff.kora.common.naming.SnakeCaseNameConverter;
import ru.tinkoff.kora.json.common.annotation.Json;

@Json
@NamingStrategy(SnakeCaseNameConverter.class)
public record DtoWithSnakeCaseNaming(String stringField, Integer integerField) {
}

File: json/json-symbol-processor/src/test/kotlin/ru/tinkoff/kora/json/ksp/dto/DtoWithSnakeCaseNaming.kt
package ru.tinkoff.kora.json.ksp.dto

import ru.tinkoff.kora.common.NamingStrategy
import ru.tinkoff.kora.common.naming.SnakeCaseNameConverter
import ru.tinkoff.kora.json.common.annotation.Json
import java.util.*

@Json
@NamingStrategy(SnakeCaseNameConverter::class)
data class DtoWithSnakeCaseNaming(val stringField: String, val integerField: Int)
